source/application/SXRMB/SXRMBAppController.cpp

```diff
@@ -348,6 +348,8 @@ void SXRMBAppController::setupExporterOptions()
 
 void SXRMBAppController::setupUserInterface()
 {
+	SXRMBBeamline *sxrmbBl = SXRMBBeamline::sxrmb();
```

Comment:
Initializing the beamline object happens in startup() for BioXAS. Just checking--is this different for SXRMB?